.ci/atime/tests.R

```diff
@@ -120,23 +120,21 @@ test.list <- atime::atime_test_list(
       data.table:::setattr(L, "class", NULL)
       data.table:::setDT(L)
     },
-    Slow = "c4a2085e35689a108d67dacb2f8261e4964d7e12", # Parent of the first commit in the PR that fixes the issue (https://github.com/Rdatatable/data.table/commit/7cc4da4c1c8e568f655ab5167922dcdb75953801)
-    Fast = "1872f473b20fdcddc5c1b35d79fe9229cd9a1d15"), # Last commit in the PR that fixes the issue (https://github.com/Rdatatable/data.table/pull/5427/commits)
+    Slow = "c4a2085e35689a108d67dacb2f8261e4964d7e12", # Parent of the first commit (https://github.com/Rdatatable/data.table/commit/7cc4da4c1c8e568f655ab5167922dcdb75953801) in the PR (https://github.com/Rdatatable/data.table/pull/5427/commits) that fixes the issue
+    Fast = "1872f473b20fdcddc5c1b35d79fe9229cd9a1d15"), # Antepenultimate commit in the PR (https://github.com/Rdatatable/data.table/pull/5427/commits) that fixes the issue
```

Comment:
I don't understand Antepenultimate, why did you change from the simpler "last" ?

Reply:
It's not really the last commit (you can check the link and verify), but I agree that it's better to avoid such a jargon - will 'third-last' work? (since that seems to be the case)

Reply:
The SHA is not associated with the last commit (you can check the link and verify), but I agree that it's better to avoid such a jargon - will 'third-last' or 'close-to-last' work?

Reply:
Is there something wrong with the last SHA in the PR, then just "last commit" will be correct? otherwise, third-to-last seems best to me (as much as I hate wasting such a perfectly valid word!)

Reply:
Great point! No issues with the SHA, just didn't get updated at the end of #6094 unfortunately (I should've tried it with the last one before changing the comment)

Thank you both!